### PR TITLE
Add java based kafka schema registry example with avro serialisation

### PR DESCRIPTION
**Pull request by snuyanzin**
# Update this heading, and describe your changes
This is a kafka schema registry java based example as part of https://github.com/aiven/devportal/pull/480 

The idea of example is the next one:
There are Kafka producer and Kafka consumer which use Avro's schema defined in `ClickRecord.avsc`
Avro schema could be automatically recompiled by maven.

Producer can generate messages based on input csv file, serializing to Avro
Consumer reads all the messages from the specified topic as ClickRecord objects deserializing from Avro

Can be build with maven, in case maven is absent it will download it
Could be built and executed with jdk 1.8+ on Linux/Mac/Windows

P.S.
This a bit reworked example, taken from https://help.aiven.io/en/articles/2302613-using-schema-registry-with-aiven-for-apache-kafka

### CODE REVIEW

**Comment by snuyanzin** [comment]
Hi @RyanSkraba, since you are one of Avro's people ;) could you please help with review here?

File: solutions/kafka-schema-registry/README.md
Comment (RyanSkraba) [inline solutions/kafka-schema-registry/README.md:4]: These should be bullet points or separate paragraphs (otherwise they appear on one line).  Same with `Preparation` below.

**Review by RyanSkraba — APPROVED** [review]
I left some minor fixes that should be corrected before merging, but all of the content LGTM!

File: solutions/kafka-schema-registry/mvnw
Comment (snuyanzin) [reply solutions/kafka-schema-registry/mvnw:1]: thanks, I added this
I only excluded `.mvn/wrapper/maven-wrapper.jar` as it could be automatically downloaded in case it is absent

**Review by RyanSkraba — APPROVED** [review]
This LGTM but requires someone with merge permissions to review!  @jlprat can you take a look?

**Comment by jlprat** [comment]
I guess we need to add the @aiven/aiven-open-source team as write access for this repo.

**Comment by jlprat** [comment]
Now it should work correctly

**Pull request merged**
